core/src/main/scala/spinal/core/internals/VerilogBase.scala

```diff
@@ -106,17 +106,31 @@ trait VerilogBase extends VhdlVerilogBase{
   }
 
   def emitEnumLiteral[T <: SpinalEnum](enum: SpinalEnumElement[T], encoding: SpinalEnumEncoding, prefix: String = "`"): String = {
-    prefix + globalPrefix + enum.spinalEnum.getName() + "_" + encoding.getName() + "_" + enum.getName()
+//    prefix + enum.spinalEnum.getName() + "_" + encoding.getName() + "_" + enum.getName()
```

Comment:
Here, and also in the other functions of this modified file, globalPrefix seems to be dropped.

what globalPrefix does, is it allow to add some prefix in the front of everything, to avoid name conflicts if you generate multiple SpinalHDL netlist and use them together in a simulation / synthesis.

Would it be possible to bring that feature back ?

Reply:
yeah, I fixed the dropped.